src/JMS/Serializer/Handler/DateHandler.php

```diff
@@ -173,6 +191,18 @@ private function parseDateTime($data, array $type, $immutable = false)
         return $datetime;
     }
 
+    private function parseDateInterval($data)
+    {
+        $dateInterval = null;
+        try {
+            $dateInterval = new \DateInterval($data);
+        } catch (\Exception $e) {
+            throw new RuntimeException(sprintf('Invalid dateinterval "%s", expected ISO 8601 format', $data));
```

Comment:
what about adding the previous exception (`$e`) as third argument of `RuntimeException` ?